local/docker-compose.yaml

```diff
@@ -35,127 +36,70 @@ services:
+        env_file:
+            -   ./tractus-x-edc/config/customer/puris-backend.properties
+        networks:
+            - miw-net
+        extra_hosts:
+            - "host.docker.internal:host-gateway" # Adjusts container's host file to allow for communication with docker-host machine
 
     edc-customer-control-plane:
+        restart: on-failure
         extends:
             file: ./tractus-x-edc/docker-compose.yaml
             service: control-plane
         container_name: customer-control-plane
-        depends_on: 
-            omejdn-daps:
-                condition: service_started # has no health-check
-            vault:
-                condition: service_started # has no health-check, needs to run script to put secrets
-            postgres:
-                condition: service_started # has no health-check, needs to run db script to create edc tables
-        environment:
-            edc.vault.hashicorp.token: ${VAULT_DEV_ROOT_TOKEN_ID}
-            edc.api.auth.key: ${EDC_API_PW}
-            edc.datasource.asset.user: ${PG_USER}
-            edc.datasource.asset.password: ${PG_PW}
-            edc.datasource.contractdefinition.user: ${PG_USER}
-            edc.datasource.contractdefinition.password: ${PG_PW}
-            edc.datasource.contractnegotiation.user: ${PG_USER}
-            edc.datasource.contractnegotiation.password: ${PG_PW}
-            edc.datasource.policy.user: ${PG_USER}
-            edc.datasource.policy.password: ${PG_PW}
-            edc.datasource.transferprocess.user: ${PG_USER}
-            edc.datasource.transferprocess.password: ${PG_PW}
+        env_file:
+            -   ./tractus-x-edc/config/customer/control-plane.properties
         ports:
-            - 8180:8180
-            - 8181:8181
-            - 8182:8182
-            - 8183:8183
-            - 8184:8184
-        volumes:
-            - ./tractus-x-edc/config/customer/control-plane.properties:/app/configuration.properties
+            - "8180:8180"
+            - "8181:8181"
+            - "8182:8182"
+            - "8183:8183"
+            - "8184:8184"
+        networks:
+            - miw-net
+        extra_hosts:
+            - "host.docker.internal:host-gateway" # Adjusts container's host file to allow for communication with docker-host machine
 
     edc-customer-data-plane:
+        restart: on-failure
         extends:
             file: ./tractus-x-edc/docker-compose.yaml
             service: data-plane
         container_name: customer-data-plane
-        depends_on:
-            omejdn-daps:
-                condition: service_started # has no health-check
-            vault:
-                condition: service_started # has no health-check, needs to run script to put secrets
-            postgres:
-                condition: service_started # has no health-check, needs to run db script to create edc tables
         ports:
-            - 8280:8280
-            - 8285:8285
+            - "8280:8280"
```

Comment:
I quickly checked for the medium issue that the container traffic is not bound to host interface. Following this [thread ](https://stackoverflow.com/questions/45109398/how-can-i-make-docker-compose-bind-the-containers-only-on-defined-network-instea) we could fix it by either
- setting ports to localhost binding: - "127.0.0.1:8280:8280" (as it's the outward facing part I guess our service names in the docker network are still valid) OR
- we can configure the networks to bind to a port at all

Could you have a look at that?

Reply:
Yes, good point. Did that with the prefix